### DISCUSSION TITLE
mobile support or Responsive?

### DISCUSSION THREAD

**Discussion by devCluna** [gilbarbara/react-spotify-web-playback#89] [Q&A] [answered]
Hi @gilbarbara. I have to acknowledge your great effort on this project.

I was wondering if there is any way to customize the SpotifyPlayer component to be responsive on mobile devices.

**Full Size Screen COOL**
![imagen](https://user-images.githubusercontent.com/39814718/147393002-563eb04c-f925-44fb-9375-2827b3369014.png)

**Mobile Size Break** (Volume Control Disapears)
![imagen](https://user-images.githubusercontent.com/39814718/147393087-8d6a81cd-9a0e-4c12-a31a-dd7ac43cf60c.png)

Also I could realize that it does not play a track in a mobile browser (Google Chrome Android). There is some way that I can reproduce a track from the mobile browser.

Thanks in advance.

**Comment by gilbarbara** [comment, marked as answer]
hey @devCluna 
Thanks!

No, you can't customize the UI. This package is just a UI interface for Spotify's Web Playback after all.

The volume control is hidden on mobile since there are dedicated hardware buttons for it on the device...

What version are you using? Mobile [support](https://developer.spotify.com/documentation/web-playback-sdk/#supported-browsers) is available since [0.9.0](https://github.com/gilbarbara/react-spotify-web-playback/releases/tag/v0.9.0)
